content/posts/newsletter-014/index.md

```diff
@@ -254,6 +278,18 @@ including a tutorial on how to get started.
 
 ## Library & Tooling Updates
 
+### [Bracket-Lib]
+
+[bracket-lib] (previously `rltk_rs`) by [@blackfuture]
+is a Rust implementation of [C++ Roguelike Toolkit][rltk-cpp].
```

Comment:
The URLs for these links are also missing.

Reply:
The link issues you found should be resolved. Sorry about that!